### DISCUSSION TITLE
How can we block fields in readonly mode getting the focus per tab key?

### DISCUSSION THREAD

**Discussion by UBK-RE** [quasarframework/quasar#14717] [Quasar App Extensions] [answered]
How can we block fields in readonly mode getting the focus per tab key? We would like to jump to the next field which is NOT in readonly mode.
Any ideas?

**Comment by MilosPaunovic** [comment, marked as answer]
https://codepen.io/milospaunovic/pen/dyKPbvp?editors=1010

**Reply by UBK-RE** [reply]
Thank you so much that will help
